module-vfs/logdump/logdump.cpp

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) 2017-2021, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#include <logdump/logdump.h>
+#include <purefs/filesystem_paths.hpp>
+#include <Logger.hpp>
+
+int dumpLogs()
+{
+    const auto logPath = purefs::dir::getLogsPath() / LOG_FILE_NAME;
```

Comment:
const lvalues are not moveable. Copy constructor will be called.

Reply:
ctrl + c, ctrl + v.
Fixed